src/manage_nsfs/manage_nsfs_cli_utils.js

```diff
@@ -109,18 +124,26 @@ async function get_config_data_if_exists(config_root_backend, config_file_path,
 /**
  * get_bucket_owner_account will return the account of the bucket_owner
  * otherwise it would throw an error
- * @param {object} global_config
- * @param {string} bucket_owner
+ * @param {Object} global_config
+ * @param {string} dir_path directory with account file (either accounts, root_accounts or access_keys)
+ * @param {string} root_account_identifier root account file name, either name or id
+ * @param {boolean} is_symlink whether while is symlink or not. Name -> true, id -> false/undef
```

Comment:
I think there is a typo here (extra "while")
```suggestion
 * @param {boolean} is_symlink whether is symlink or not: name -> true, id -> false/undefined
```